packages/dialect-react-ui/components/Notifications/index.tsx

```diff
@@ -85,38 +90,52 @@ function CreateThread() {
   );
 }
 
-function Settings(props: { toggleSettings: () => void }) {
+function Settings(props: {
+  toggleSettings: () => void;
+  notifications: NotificationType[];
+}) {
   const { dialectAddress, deleteDialect, isDialectDeleting, deletionError } =
     useDialect();
   const { colors, textStyles, icons } = useTheme();
 
   return (
     <>
       <div className="mb-3">
-        <p className={cs(textStyles.body, 'mb-1')}>Included event types</p>
-        <ul className={cs(textStyles.bigText, 'list-disc pl-6')}>
-          <li>Welcome message on thread creation</li>
-        </ul>
+        <h2 className={cs(textStyles.h2, 'mb-1')}>Notifications</h2>
+        {props.notifications?.map((type) => (
+          <ValueRow
+            key={type.name}
+            label={type.name}
+            highlighted
```

Comment:
I see that almost all ValueRow components now have `highlighted` passed as a prop. Does it make sense to make it a default?
Also, this might (saying "might" because we have to careful) cause a "boolean trap" since it's a styling feature and if we might have variations like "highlighted" "dimmed" "normal" and the conflict between each other (I've only noticed this sorry...)
Not saying that we need to change this now, but please. leave a todo to improve this. Thanks!